src/node_tests/ndt_client.js

```diff
@@ -226,7 +257,8 @@ function ndt_c2s_test() {
         }
         if (state === "WAIT_FOR_TEST_FINALIZE" && type === TEST_FINALIZE) {
             state = "DONE";
-            console.log("C2S rate: ", 8 * TRANSMITTED_BYTES / 1000 / (test_end - test_start));
+            console.log("C2S test complete using " + url_protocol);
```

Comment:
No, we give one success message for each test unconditionally.  Otherwise, when the tests run you can't tell whether any progress is being made.